File: BackendAndFrontend/Conit/Conit.DAL/Entities/Company.cs
using System;

namespace Conit.DAL.Entities
{
    public class Company
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string PictureId { get; set; }

        public string PhoneNumber { get; set; }

        public string ContactName { get; set; }

        public DateTime DateOfAdding { get; set; }

        public string AspNetUserId { get; set; }

        public bool IsDeleted { get; set; }
    }
}

File: Mobile/flutter_app/flutter_app/lib/company.dart
class Company {
  final int id;
  final String name;
  final String description;
  final String contactName;
  final String phoneNumber;
  final String pictureId;

  Company._({
    this.id,
    this.name,
    this.description,
    this.contactName,
    this.phoneNumber,
    this.pictureId
});

  factory Company.fromJson(Map<String, dynamic> json) {
    return new Company._(
      id: json['id'],
      name: json['name'],
      description: json['description'],
      contactName: json['contactName'],
      phoneNumber: json['phoneNumber'],
      pictureId: json['pictureId']
    );
  }
}